VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/wr-participant-list-shared.component.ts

```diff
@@ -104,9 +105,50 @@ export abstract class WRParticipantStatusListDirective {
     }
 
     protected filterNonJudgeParticipants(): void {
-        this.nonJudgeParticipants = this.conference.participants.filter(
+        const nonJudgeParts = this.conference.participants.filter(
             x => x.role !== Role.Judge && x.role !== Role.JudicialOfficeHolder && x.hearing_role !== HearingRole.OBSERVER
         );
+
+        const individualWithInterpreterIndex = nonJudgeParts.findIndex(
+            x =>
+                x.role === Role.Individual &&
+                x.hearing_role !== HearingRole.INTERPRETER &&
+                Array.isArray(x.linked_participants) &&
+                x.linked_participants.length > 0
+        );
+        if (individualWithInterpreterIndex === -1) {
+            this.nonJudgeParticipants = nonJudgeParts;
+        } else {
+            this.nonJudgeParticipants = this.orderForInterpreter(nonJudgeParts, individualWithInterpreterIndex);
+        }
+    }
+
+    hasInterpreterLink(participant: ParticipantResponse) {
+        return participant.linked_participants.some(x => x.link_type === LinkType.Interpreter);
+    }
+
+    isInterpreter = function (participant: ParticipantResponse) {
```

Comment:
why does this one use the function keyword?

Reply:
because I was trying to call it from the *ngIf then reverted to the expression method

Reply:
fixed